src/modules/fetcher/__mocks__/NativeFetcherModule.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import type { FetchResultType } from '../FetcherModule'
+import { NativeEventEmitter } from 'react-native'
+
+export type NativeFetcherModuleType = {
```

Comment:
I think this type is not necessary.  If it is then use it directly in this file in line 12:

`const NativeFetcherModule:  NativeFetcherModuleType = {`